messaging/MessagingExampleSwift/AppDelegate.swift

```diff
@@ -91,6 +91,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     // Print full message.
     print(userInfo)
+
+    if completionHandler {
```

Comment:
It does not, you are correct here Ian, however seems like an unnecessary check since this is called by the OS, the handler should always be available.